sdcm/sct_events/filters.py

```diff
@@ -46,6 +47,11 @@ def eval_filter(self, event: LogEventProtocol) -> bool:
         if self.filter_line:
             result &= self.filter_line in (getattr(event, "line", "") or "")
 
+        if self.filter_regex:
+            line_content = getattr(event, "line", "") or ""
+            if not self.filter_regex.search(line_content):
+                result = False
```

Comment:
I would overwrite the result, it can yield incorrect results like that
```suggestion
        if self._regex:
            result &= self._regex.match(str(event)) is not None
```